ARCH-1205 - No default commitish (#8)
im-open/create-release: README.md

- [License](#license)

## Inputs
| Parameter                 | Is Required                      | Description                                                                                                                                |
| ------------------------- | -------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
| `github-token`            | true                             | A token with permission to create and delete releases.  Generally secrets.GITHUB_TOKEN.                                                    |
| `tag-name`                | true                             | The name of the tag.                                                                                                                       |
| `release-name`            | false                            | The name of the release. Defaults to the tag name if not provided.                                                                         |

> | `commitish`               | true                             | Specifies the commitish value that determines where the Git tag is created from. Can be any branch or commit SHA.                          |

Reviewer: ```suggestion
| `commitish`               | true                             | Specifies the commitish value that identifies the commit to tag. Can be any branch or commit SHA.                          |
```

          dotnet publish -c Release -o ./published_app 
          (cd published_app && zip -r ../${{env.DEPLOY_ZIP}} .)

      - name: Create Release
        id: create_release
        uses: im-open/create-release@v3.0.0
        with:
          github-token: ${{ secrets.GITHUB_TOKEN }}
          tag-name: ${{ steps.version.outputs.VERSION }}
          commitish: ${{ env.REF_TO_BUILD }}
          prerelease: ${{ env.IS_PRERELEASE }}
          asset-path: ${{ env.PROJECT_ROOT }}/${{ env.DEPLOY_ZIP }}
          asset-name: ${{ env.DEPLOY_ZIP }}
          asset-content-type: application/zip
          # The release might already exist if you hit 're-run jobs' on a workflow run that already completed
          # once. Creating a release when one already exists will fail, add the arg here to just delete it.

>           delete-existing-release: true

Reviewer: Yeah. I like the example!

I wonder about adding an `if` somewhere to avoid building closed PR that aren't merged. But maybe that would add noise and detract from the focus of the example?

Author: I went back and forth about doing that.  It would make the example clearer, but it's also adding one more thing to the example that wasn't necessarily the point of it.

Author: I added the example for clarity.
